source/Applications/Advanced/MultiCamera/README.md

```diff
@@ -1,5 +1,10 @@
 # Multi Camera
 
+To use multiple Zivid Two cameras, set custom IP configuration. See guide for [multiple zivid cameras](https://support.zivid.com/latest/academy/camera/multiple-zivid-cameras.html?highlight=multi%20camera) in our Knowledge Base for complete guide. 
```

Comment:
"?highlight=multi%20camera" do we really want to highlight this text in the browser?

Reply:
hehe we probably dont want that

Reply:
Changed the link to ip config page instead.